addon/helpers/format-number.js

```diff
@@ -1,51 +1,30 @@
 import { helper as buildHelper } from '@ember/component/helper';
 
-var formats = [
-  {
-    value: 1e3,
-    sym: 'k'
-  },
-  {
-    value: 1e6,
-    sym: 'M'
-  },
-  {
-    value: 1e9,
-    sym: 'B'
-  }
-];
+export const PREVENT_COMPACT_NOTATION_BELOW = 1000;
+export const ROUND_TO_INTEGER_ABOVE = 100;
 
-var formatNumber = function formatNumber(params) {
-  var number = params[0];
+const _getFormatter = function (number) {
```

Comment:
constant format is all over the place, outsite of a class definition, we usually have uppercase for constants - without `_` a the beginning. 
I understand that this is to mark is as private, but why arent we just using normal function notations here ?

```
function getFormat(number) {
  // code
}

export function formatNumber(params) {
   // code
}
```

Reply:
@phndiaye wdyt ?